src/templates/Article/docs.md

```diff
@@ -791,6 +799,47 @@ Zweitens: ich habe erklärt mit diese zwei Spieler: nach Dortmund brauchen viell
 
 <hr /></section>
 
+Denn esse est percipi - Sein ist wahrgenommen werden.
+
+<hr /></section>
+`}</Markdown>
+```
+
+### Edge-to-Edge
+
+Simpy unwrap from center
```

Comment:
Typo: Simpy -> Simply
(also: maybe `detach` instead of `unwrap`)

Reply:
Thx for the review. `unwrap` is a slate term so leaving it at that.